### PR TITLE
Update project TypeScript

### PR DESCRIPTION
**Pull request by klzns**
#### What does this PR do? \*

Fix some undefined references:

```
Cannot read property 'sellers' of undefined
```

#### How to test it? \*

https://vysk--n95.myvtex.com/test-product-10970/p?skuId=1096212

### CODE REVIEW

File: react/SellerBodyCell.tsx [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
 import { useCssHandles } from 'vtex.css-handles'
 
 const SELLERS_CSS_HANDLES = ['sellerBodyCell'] as const
 
-const SellerBodyCell: StorefrontFunctionComponent = ({ children }) => {
+interface Props {
+  children: ReactNode
+}
+
+function SellerBodyCell({ children }: Props) {
```
Comment (victorhmp) [inline react/SellerBodyCell.tsx:11]: You could just use `PropsWithChildren`, although it will look weird 😂

File: react/SellerHead.tsx [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
 import { useCssHandles } from 'vtex.css-handles'
 
 const SELLERS_CSS_HANDLES = ['sellerHead'] as const
 
-const SellerHead: StorefrontFunctionComponent = ({ children }) => {
+interface Props {
+  children: ReactNode
+}
+
+function SellerHead({ children }: Props) {
```
Comment (victorhmp) [inline react/SellerHead.tsx:11]: Same thing here

File: react/SellerRow.tsx [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
 import { useCssHandles } from 'vtex.css-handles'
 
 const SELLERS_CSS_HANDLES = ['sellerRow'] as const
 
-const SellerRow: StorefrontFunctionComponent = ({ children }) => {
+interface Props {
+  children: ReactNode
+}
+
+function SellerRow({ children }: Props) {
```
Comment (victorhmp) [inline react/SellerRow.tsx:11]: Same

**Review by victorhmp — APPROVED** [review]

File: react/SellerBody.tsx [hunk 4]
```diff
@@ -1,23 +1,28 @@
...
 import { CurrentSellerProvider } from './CurrentSellerContext'
 
 const SELLERS_CSS_HANDLES = ['sellerList']
 
-const SellerBody: StorefrontFunctionComponent = ({ children }) => {
+interface Props {
+  children: ReactNode
+}
+
+function SellerBody({ children }: Props) {
```
Comment (igorbrasileiro) [inline react/SellerBody.tsx:15]: What do you think about use `PropsWithChildren<{}>`?

File: react/SimulateShipping.tsx [hunk 5]
```diff
@@ -111,10 +113,4 @@ const SimulateShipping: FC = () => {
   )
 }
 
-interface ShippingItem {
```
Comment (igorbrasileiro) [inline react/SimulateShipping.tsx:114]: Why remove this interface?

**Review by igorbrasileiro — APPROVED** [review]
LGTM

File: react/SimulateShipping.tsx (hunk 5 above)
Comment (klzns) [reply react/SimulateShipping.tsx:114]: It's not being used anymore.

**Pull request merged**